server/src/YmlToBdl.g4

```diff
@@ -118,50 +119,41 @@ ENUM : 'enum';
 //Vérifier adéquation avec les terminaux définis par Olivier
 ID : LETTER ALPHANUM* ;
 
-//We store the whitespace in hidden channel to take back the ones around comments
-
 fragment MULTILINE_COMMENT_START: '/*';
 fragment MULTILINE_COMMENT_END: '*/';
 
-LINE_COMMENT : '//' ~[\n\r]* ;
-MULTILINE_COMMENT: MULTILINE_COMMENT_START .*? MULTILINE_COMMENT_END;
+LINE_COMMENT : '//' ~[\n\r]*  -> channel(HIDDEN);

```

Comment:
Don't process the comments, but put them in a special channel to be able to do something from them in the future.